docs/docs/09-rfcs/2023-03-08-wing-preview-environment.md

```diff
@@ -0,0 +1,364 @@
+2. Once singed in, you will be redirected back to the preview environment URL.
+
+#### Wing Cloud Dashboard for Signed-In Users (#3)
+
+1. Once signed in, you will be able to navigate directly to https://wing.cloud/dashboard.
+2. You can see and manage all your projects and teams.
+3. You will see all preview environments associated with your personal account and teams you are member in.
+4. In order to see a preview environment, just click on the preview environment URL from the list of all your preview environments.
+5. For each preview environment, you can see the deployment logs and the test results and logs.
+
+#### Create a new personal Project (#4)
+
+1. To create a new project, click on "New Project" under "Personal Account".
+2. Choose between the GitHub repositories that you have access to.
+3. Define your secret values.
+4. Make sure there's a `main.w` file in the root of the repository.
+5. Click on "Deploy".
+6. You'll be redirected to your new project's page (https://wing.cloud/dashboard/-user/-project).
+
+#### Create a new Team (#5)
+
+1. Click on "New Team".
+2. Write a team name.
+3. Optionally, invite other Wing Cloud users to the team (by GitHub user ID).
+
+#### Create a new team Project (#6)
+
+1. Click on "New Project" under the team of your choice.
+2. Choose between the GitHub repositories that you have access to.
+3. Define your secret values.
+4. Make sure there's a `main.w` file in the root of the repository.
+5. Click on "Deploy".
+6. You'll be redirected to your new project's page (https://wing.cloud/dashboard/-team/-project).
+
+#### Preview Environment for production branch (#7)
+
+For the repository production branch, there is a single preview environment that will constantly be up-to-date with the latest code committed.
+A link to the preview environment is available in the repository main page ("About" section).
+Production branch environment url structure is: `https://wing.cloud/-account/dashboard/-project/-branch`
+Once the installation of the Wing Cloud application is completed, an initial preview environment will be created and will be available in the repository main page.
+
+#### Preview Environments for Pull Requests (#8)
+
+Upon each creation of a pull request an automatic comment will be added to the PR and will guide the developer to a dedicated preview environment.
+Provide a streamlined process for updating a pull request preview environment, each code changes in a PR will redeploy the preview environment and will also:
+
+1. Provide a real-time build and deployment status updates
+2. While preview environment is being updated, the user that is currently using the environment will be notified and will be able to refresh the page to get the latest changes.
+3. Ensure a consistent and unique url for each preview environment. The url structure: `https://wing.cloud/dashboard/-account/-project/-branch` (account is a <user> or a <team>)
```

Comment:
```suggestion
3. Ensure a consistent and unique url for each preview environment. The url structure: `https://wing.cloud/-account/-project/-branch` (account is a <user> or a <team>)
```

If we really need another URL component, it should be `/environments` or `/projects`